### PR TITLE
added / fixed websites

### LINKED ISSUE #3087: [Site Request] comicfx
comicfx.net

Site: https://comicfx.net/
Manga List: https://comicfx.net/komik-list
Manga example: https://comicfx.net/komik/the-nine-heaven-of-martial-arts
Chapter online viewer example: https://comicfx.net/komik/the-nine-heaven-of-martial-arts/Chapter-151/1

Languages
Indonesia

### LINKED ISSUE #3100: [Site Request] Iris Scanlator
**Name of the website**
Iris Scanlator

**Website urls**
- Site: https://irisscanlator.com/
- Manga List: https://irisscanlator.com/manga/
- Manga example: https://irisscanlator.com/manga/owari-no-seraph/
- Chapter online viewer example: 
- - https://irisscanlator.com/manga/owari-no-seraph/volume-1/capitulo-1/
- - https://irisscanlator.com/manga/owari-no-seraph/capitulo-82/

**Languages**
Portuguese

**Template**
WordpressMadara

### LINKED ISSUE #3101: [Site Request] Sindria Scanlator
**Name of the website**
Sindria Scanlator

**Website urls**
- Site: https://sindriascanlator.com/
- Manga List: https://sindriascanlator.com/mangas/
- Manga example: https://sindriascanlator.com/mangas/maid-no-kishi-san/
- Chapter online viewer example: https://sindriascanlator.com/mangas/maid-no-kishi-san/capitulo-01/

**Languages**
Portuguese

**Template**
WordpressMadara

### LINKED ISSUE #3102: [Site Request] Mode Scanlator
**Name of the website**
Mode Scanlator

**Website urls**
- Site: https://modescanlator.com/
- Manga List: https://modescanlator.com/manga/
- Manga example: https://modescanlator.com/manga/doupo-cangqiong/
- Chapter online viewer example: https://modescanlator.com/manga/doupo-cangqiong/capitulo-323/

**Languages**
Portuguese

**Template**
WordpressMadara

### LINKED ISSUE #3106: [Site Request] mangaweebs
mangaweebs

**Website urls (examples below)**
- Site: https://mangaweebs.in/

- Manga List: https://mangaweebs.in/manga/the-second-coming-of-gluttony/

- Chapter online viewer example: https://mangaweebs.in/manga/the-second-coming-of-gluttony/chapter-1/

**Languages**
Engelish

**Website relationship*
none

### LINKED ISSUE #3109: [Site Request] Olho da Lua
**Name of the website**
Olho da Lua

**Website urls**
- Site: https://olhodalua.xyz/
- Manga List: https://olhodalua.xyz/manga/
- Manga example: https://olhodalua.xyz/manga/xian-ni/
- Chapter online viewer example: https://olhodalua.xyz/manga/xian-ni/01/

**Languages**
Portuguese

**Template**
WordPressMadara

### LINKED ISSUE #3110: [Site Request] AnimaRegia
**Name of the website**
AnimaRegia

**Website urls**
- Site: https://animaregia.net/
- Manga List: https://animaregia.net/manga-list/
- Manga example: https://animaregia.net/manga/out-ptbr/
- Chapter online viewer example: https://animaregia.net/manga/out-ptbr/01/

**Languages**
Portuguese

**Template**
MangaReaderCMS

### PR DESCRIPTION
**Pull request by 27136773**

### CODE REVIEW

**Comment by Slowftw** [comment]
😳🙏

File: src/web/mjs/connectors/TonariNoYoungJump.mjs [hunk 1]
```diff
@@ -8,6 +8,7 @@ export default class TonariNoYoungJump extends CoreView {
         super.label = 'となりのヤングジャンプ (Tonari no Young Jump)';
         this.tags = [ 'manga', 'japanese' ];
         this.url = 'https://tonarinoyj.jp';
+        this.requestOptions.headers.set('x-user-agent', 'Mozilla/5.0 (Linux; Android 9; Pixel) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/85.0.4026.0 Mobile Safari/537.36');
```
Comment (Sheepux) [inline src/web/mjs/connectors/TonariNoYoungJump.mjs]: Handled at coreview template level by PR #3123

Comment (27136773) [reply src/web/mjs/connectors/TonariNoYoungJump.mjs]: Reverted cange

**Review by Sheepux — APPROVED** [review]
/LGTM

**Pull request merged**